### DISCUSSION TITLE
seeking help with unicode symbols

### DISCUSSION THREAD

**Discussion by KBLiveSolutions** [Bodmer/TFT_eSPI#3269] [Q&A - Fonts]
I need to show symbols such as ▶, ⇉, ❷, ⌻, ∅, ☆ ... to show a few. They all are unicode characters.
I have tried to do look for a custom font that would have these symbols but didn't find anything. 
I have tried to create my own font but didn't succeed.
Does anyone have a clue on how I can achieve that ? I'm stuck right now.
Thanks !

**Comment by TigoTas** [comment]
Did you find any solutions for your matter? I also need to print symbols such as "arrows" and "triangles", like those [here](https://www.compart.com/en/unicode/U+25B2).

Does any of the available fonts have these unicode symbols?
